portality/templates/dashboard/index.html

```diff
@@ -1,41 +1,78 @@
 {% extends "layouts/dashboard_base.html" %}
 {# ~~Dashboard:Page~~ #}
 {% set TODOS = {
-        constants.TODO_MANED_STALLED: "Chase <strong>stalled</strong> application",
-        constants.TODO_MANED_FOLLOW_UP_OLD: "Follow up with <strong>old<strong> application",
-        constants.TODO_MANED_READY: "Make a decision on <strong>ready<strong> application",
-        constants.TODO_MANED_COMPLETED: "Review <strong>completed<strong> application",
-        constants.TODO_MANED_ASSIGN_PENDING: "Assign <strong>pending<strong> application"
+        constants.TODO_MANED_STALLED: {
+            "text" : "Chase a <strong>stalled</strong> application",
+            "colour" : "var(--salmon)",
+            "feather": "coffee"
+        },
+        constants.TODO_MANED_FOLLOW_UP_OLD: {
+            "text": "Follow up with an <strong>old</strong> application",
+            "colour" : "var(--sanguine)",
+            "feather": "clock"
+        },
+        constants.TODO_MANED_READY: {
+            "text" : "Make a decision on a <strong>ready</strong> application",
+            "colour" : "var(--dark-green)",
+            "feather": "thumbs-up"
+        },
+        constants.TODO_MANED_COMPLETED: {
+            "text" : "Review a <strong>completed</strong> application",
+            "colour" : "var(--mid-green)",
+            "feather": "check-circle"
+        },
+        constants.TODO_MANED_ASSIGN_PENDING: {
+            "text" : "Assign a <strong>pending</strong> application",
+            "colour" : "var(--yellow)",
+            "feather": "inbox"
+        }
     }
 %}
 
 {% block content %}
 <main>
-
   <h1>Hi, {{ current_user.name }}!</h1>
 
   <section>
-    <h2 class="label">Here are your priorities:</h2>
-    <ol class="unstyled-list">
+    <h2 class="type-04">Here are your priorities ({{ todos|length }}):</h2>
```

Comment:
I'm not sure you can use `todos|length` here, as it's not that meaningful.  It doesn't tell you how many todos you have, it just tells you how many are in the list on the page, which is a function of the number we've configured it to show.

Reply:
Oh I didn’t realise that! Is there a way to show the number of TODOs?

Reply:
There is not, see some information about why buried in my comments here: https://github.com/DOAJ/doajPM/issues/3176#issuecomment-1018440621 and https://github.com/DOAJ/doajPM/issues/3176#issuecomment-1023070140